### DISCUSSION TITLE
Cleanup ?

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#352] [General]
Might be nice to have a clean up option for obsolete directories

![image](https://github.com/ZOSOpenTools/meta/assets/42328411/1dc79b8c-61eb-4566-96ba-ec04af97b666)

Not sure how but it would be nice.

**Comment by v1gnesh** [comment]
The cleanup script could delete all non-symlink'd directories.. :)

**Comment by MikeFultonDev** [comment]
My hunch is this will get 'fixed' as part of the next iteration of zopen setup.

**Comment by MikeFultonDev** [comment]
@lbdyck There are 'remove' and 'clean' verbs now. Are these adequate?
